test/types/tuple/split-init/tupleDestructure.chpl

```diff
@@ -0,0 +1,3 @@
+var (a, b);
+(a, b) = (1, "hi");
+writeln((a, b));
```

Comment:
This and many other files in this PR seem to be missing a newline at the end of the file. Not sure it matters much but GitHub draws a red icon...